### ISSUE TITLE
[Sonos] tuneinstationid not updated

### ISSUE THREAD

**Issue by Battle92** [openhab/openhab-addons#14794]
## Expected Behavior
tunein stationid should change as soon as a tunein sender is changed. but it doesn't. the station id always remains the same.

## Current Behavior

tuneinstationid not change. it always remains the same.

no logs in the openhab log viewer

the tuneinstationid should change as soon as a station is changed.

## Possible Solution
-

## Steps to Reproduce (for Bugs)

image of the item: https://freeimage.host/i/Hvuj9l2

## Context
i am trying to determine the changing tuneinstationids 

## Your Environment

openHAB 3.4.2
Sonos Binding 3.4.2

**Comment by lolodomo** [comment]
I can't reproduce your problem. Look at this:
```

2023-05-01 23:40:23.228 [INFO ] [openhab.event.ItemStateChangedEvent ] - Item 'SonosBureauRadio' changed from 24940 to 16699
2023-05-01 23:40:58.317 [INFO ] [openhab.event.ItemStateChangedEvent ] - Item 'SonosBureauRadio' changed from 16699 to 35951
2023-05-01 23:41:12.369 [INFO ] [openhab.event.ItemStateChangedEvent ] - Item 'SonosBureauRadio' changed from 35951 to 22837
2023-05-01 23:41:48.347 [INFO ] [openhab.event.ItemStateChangedEvent ] - Item 'SonosBureauRadio' changed from 22837 to 28957
2023-05-01 23:42:21.878 [INFO ] [openhab.event.ItemStateChangedEvent ] - Item 'SonosBureauRadio' changed from 28957 to 27729
2023-05-01 23:42:32.133 [INFO ] [openhab.event.ItemStateChangedEvent ] - Item 'SonosBureauRadio' changed from 27729 to 23833
2023-05-01 23:42:49.002 [INFO ] [openhab.event.ItemStateChangedEvent ] - Item 'SonosBureauRadio' changed from 23833 to 24040
2023-05-01 23:43:19.541 [INFO ] [openhab.event.ItemStateChangedEvent ] - Item 'SonosBureauRadio' changed from 24040 to 27729
2023-05-01 23:43:30.272 [INFO ] [openhab.event.ItemStateChangedEvent ] - Item 'SonosBureauRadio' changed from 27729 to 29458
2023-05-01 23:43:57.686 [INFO ] [openhab.event.ItemStateChangedEvent ] - Item 'SonosBureauRadio' changed from 29458 to 28957
2023-05-01 23:44:39.494 [INFO ] [openhab.event.ItemStateChangedEvent ] - Item 'SonosBureauRadio' changed from 28957 to 185291
2023-05-01 23:45:02.258 [INFO ] [openhab.event.ItemStateChangedEvent ] - Item 'SonosBureauRadio' changed from 185291 to 320257
2023-05-01 23:45:24.084 [INFO ] [openhab.event.ItemStateChangedEvent ] - Item 'SonosBureauRadio' changed from 320257 to 320274
2023-05-01 23:46:09.525 [INFO ] [openhab.event.ItemStateChangedEvent ] - Item 'SonosBureauRadio' changed from 320274 to 198434
2023-05-01 23:46:55.149 [INFO ] [openhab.event.ItemStateChangedEvent ] - Item 'SonosBureauRadio' changed from 198434 to 199665
```

**Comment by lolodomo** [comment]
I hope it is clear at least that this channel will be updated only when you play tuneIn.

**Comment by lolodomo** [comment]
@Battle92 : do you start radio from Alexa ? Because in this case, I fixed something: PR #14164. The fix is in OH4 but not in OH3. I am going to propose to backport it in OH3.

**Comment by lolodomo** [comment]
> The fix is in OH4 but not in OH3. I am going to propose to backport it in OH3.

Patch for Alexa should be included in OH 3.4.4.
